.github/workflows/ci.yml

```diff
@@ -178,7 +178,9 @@ jobs:
       - build
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
```

Comment:
This is the extent of the changes to our actions. This pulls in the submodules recursively, so they always reflect the latest commit to `main` every time they run.